plr--8.3.0.18.sql

```diff
@@ -50,7 +50,7 @@ CREATE OR REPLACE FUNCTION r_typenames()
 RETURNS SETOF r_typename AS '
   x <- ls(name = .GlobalEnv, pat = "OID")
   y <- vector()
-  for (i in 1:length(x)) {y[i] <- eval(parse(text = x[i]))}
+  for (i in 1:length(x)) {y[i] <- pg.reval(sprintf("`%s`", x[i]))}
```

Comment:
You can't modify plr--8.3.0.18.sql, you need to suppy a new plr--newversion.sql file, plus plr--8.3.0.18--newversion.sql.
(I suggest using the opportunity to shorten the version number to something like 8.1.)

Reply:
This does not change behavior drastically. It is just a bug fix for those like me having odd table names. If current version works, there is no need to upgrade.
It just occurred to me that the shorter way to write this stuff is `get(x[i])`. I will adjust PR